### PR TITLE
Feature/log file upload

### PR DESCRIPTION
**Pull request by cbrandt6**

### CODE REVIEW

**Review by Rithonn — DISMISSED** [review]
terrible stuff

File: swarmdjango/core/views/logViewSet.py [hunk 1]
```diff
@@ -1,12 +1,138 @@
...
+
+            # Open and begin processing the uploaded files
+                zip_root = ''
+                with ZipFile('upload.zip', 'r') as upload:
+
+                    # Extract the zip file to access the files
+                    upload.extractall()
+
+                    # The log files will be under a common 'root' directory
+                    zip_root = upload.namelist()[0]
```
Comment (Walrussuit101) [inline swarmdjango/core/views/logViewSet.py:55]: We decided to move `zip_root` and `base_dir` assignments outside the try/catch

**Review by Walrussuit101 — APPROVED** [review]
Other than the change below, it worked when uploading `Run_Allstop.zip` :+1:

**Review by Mnenmenth — APPROVED** [review]
Good stuff :horse:

**Pull request merged**